dnf-behave-tests/features/plugins-core/builddep-modularity.feature

```diff
@@ -0,0 +1,90 @@
+Feature: Tests for builddep command on modular system
+
+Background: Enable builddep plugin
+  Given I enable plugin "builddep"
+    And I use the repository "dnf-ci-fedora"
+
+# module ninja:master [d] contains ninja-build-0:1.8.2-4.module_1991+4e5efe2f.x86_64
+# ninja:legacy contains ninja-build-0:1.5.2-1.module_1991+4e5efe2f.x86_64
+# ninja:development contans ninja-build-1.9.2-1.module_1991+4e5efe2f.x86_64
```

Comment:
Maybe also add that these three module streams are from dnf-ci-fedora-modular repo, so that it's immediately clear. But it's not that important.

Reply:
Thanks for feedback, fixed.